fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1418,6 +1418,8 @@ public boolean updateBinlogConfigMeta(Database db, Long tableId, Map<String, Str
         newBinlogConfig.incVersion();
         try {
             GlobalStateMgr.getCurrentState().modifyBinlogMeta(db, olapTable, newBinlogConfig);
+            LOG.info("update binlog config of table {}, the binlog after modified is : {}",
```

Comment:
please print binlog config before modified

Reply:
the current binlog config has been printed